app/controllers/destinations_controller.rb

```diff
@@ -23,6 +23,12 @@ def create
         end
     end
 
+    def destroy
+        destination = Destination.find(params[:id])
+        destination.destroy
+        render json: destination
+    end
+
```

Comment:
how do you render what you just destroyed?  :)

Reply:
I am tracking!

Reply:
you can just use 'destinations'

Reply:
Oh yeah ! Like the first method declares